shell/src/main/java/org/apache/accumulo/shell/commands/CreateTableCommand.java

```diff
@@ -26,6 +30,7 @@
 
 import org.apache.accumulo.core.client.AccumuloException;
 import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.IteratorSetting;
```

Comment:
You can also import the IteratorScope inner enum to make code shorter.  

```java
import org.apache.accumulo.core.iterators.IteratorUtil.IteratorScope;
```

Reply:
Added import and shortened up the affected lines.